libcxx/include/fstream

```diff
@@ -245,6 +267,18 @@ public:
 #  endif
   _LIBCPP_HIDE_FROM_ABI basic_filebuf* __open(int __fd, ios_base::openmode __mode);
   basic_filebuf* close();
+#  if _LIBCPP_STD_VER >= 26
+  _LIBCPP_HIDE_FROM_ABI native_handle_type native_handle() const noexcept {
+    _LIBCPP_ASSERT_UNCATEGORIZED(this->is_open(), "File must be opened");
```

Comment:
Can runtime assertions be validated? I couldn't figure it out.

Reply:
Yes, but we don't always write tests for it. If you write a test it should be new standalone test. This test won't be executed in all configurations. https://github.com/llvm/llvm-project/blob/main/libcxx/test/libcxx/containers/sequences/array/array.zero/assert.front.pass.cpp is a nice example how to test this.